### PR TITLE
chore: Prepare for release 4.0.0

### PR DESCRIPTION
**Pull request by opti-jnguyen**
Summary
-------

-  Prepare for release 3.11.0

Test plan
---------

- Travis CI Tests

Issues
------

-  https://optimizely.atlassian.net/browse/OASIS-7933

### CODE REVIEW

**Review by Mat001 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ January 10th, 2022
 
 * For details, refer to our documentation pages: [OptimizelyUserContext](https://docs.developers.optimizely.com/full-stack/v4.0/docs/optimizelyusercontext-python) and [Forced Decision methods](https://docs.developers.optimizely.com/full-stack/v4.0/docs/forced-decision-methods-python).
 
+### Breaking:
```
Comment (jaeopt) [inline CHANGELOG.md:15]:
```suggestion
### Breaking Changes:
```

**Review by jaeopt — CHANGES_REQUESTED** [review]
looks good other than title change

File: CHANGELOG.md (hunk 1 above)
Comment (jaeopt) [reply CHANGELOG.md:15]: Don't we need to change version.py?

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**